src/streamlink/plugin/api/useragents.py

```diff
@@ -19,6 +19,6 @@
             "AppleWebKit/600.7.12 (KHTML, like Gecko) Version/8.0.7 Safari/600.7.12")
 SAFARI_7 = ("Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) "
             "AppleWebKit/537.75.14 (KHTML, like Gecko) Version/7.0.3 Safari/7046A194A")
-FIREFOX = "Mozilla/5.0 (Windows NT 10.0; WOW64; rv:50.0) Gecko/20100101 Firefox/50.0"
+FIREFOX = "Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:52.0) Gecko/20100101 Firefox/52.0"
```

Comment:
Are we going to have any issues swapping this over? I can't think of anything and all the tests pass, but it would be worth a sanity check.

Reply:
it shouldn't be an issue,

it's just an update for a still used User-Agent with Firefox
`privacy.resistFingerprinting` in Firefox57 will change the User-Agent to this

Reply:
Looks perfectly reasonable to me 😎